helmfile/upstream/cert-manager/values.yaml

```diff
@@ -396,3 +405,89 @@ cainjector:
     # annotations: {}
     # Automount API credentials for a Service Account.
     automountServiceAccountToken: true
+
+# This startupapicheck is a Helm post-install hook that waits for the webhook
```

Comment:
Shouldn't we now be able to move the installation of cert-manager from [04-cert-manager.yaml](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/04-cert-manager.yaml) to [50-applications.yaml](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/50-applications.yaml)?
Or is having `needs: cert-manager/cert-manager` in applications worse than having cert-manager in it's own statefile?

Reply:
Was able to fix this!